py/server/tests/test_table_factory.py

```diff
@@ -200,11 +200,76 @@ def test_dtw_single_string_arg(self):
 
         col_defs = {"A_Long": dtypes.long}
         table_writer = DynamicTableWriter(col_defs)
-        table_writer.write_row(10**10)
+        table_writer.write_row(10 ** 10)
         t = table_writer.table
         self.wait_ticking_table_update(t, row_count=1, timeout=5)
         self.assertIn("10000000000", t.to_string())
 
+    def test_input_table(self):
+        col_defs = {
+            "ByteArray": dtypes.byte_array,
+            "ShortArray": dtypes.short_array,
+            "Int32Array": dtypes.int32_array,
+            "LongArray": dtypes.long_array,
+            "Float32Array": dtypes.float32_array,
+            "DoubleArray": dtypes.double_array,
+            "StringArray": dtypes.string_array,
+        }
```

Comment:
Why are these tests oriented to array types instead of primitives? Nothing "wrong" with that, I just would have expected something around primitive types.